### PR TITLE
Add information about Messenger changes in ConvAPI due to new privacy…

### PR DESCRIPTION
**Pull request by Filip-Kus**
Add information about Messenger changes in ConvAPI due to new privacy rule in Europe.

### CODE REVIEW

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md [hunk 1]
```diff
@@ -145,6 +145,18 @@ Conversation API supports Facebook Messenger and allows sending messages from Fa
...
+> 🚧 Warning (until 16th December)
+>
+> Regarding the changes introduced by “Messenger” to comply with new privacy rules in Europe, delivered and read callbacks will not be shipped.
+>
+> This condition impacts you if:
+> - Your app points to messenger bot which is created for Europe page.
+> - Your app points to messenger bot which is created for a page that has European admins. 
+> - You send a message to people in Europe. 
+> 
+> Restriction should be dismissing in the future, currently “Messenger” working on it.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support/conversation-facebook-messenger.md:157]: restrictions will be revised by Facebook Messenger in the future.

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md [hunk 2]
```diff
@@ -145,6 +145,18 @@ Conversation API supports Facebook Messenger and allows sending messages from Fa
 
 - You should receive callbacks with information that message has been delivered and read. The channel is now configured.
 
+> 🚧 Warning (until 16th December)
+>
+> Regarding the changes introduced by “Messenger” to comply with new privacy rules in Europe, delivered and read callbacks will not be shipped.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support/conversation-facebook-messenger.md:150]: I think Christopher asked Tauben lately, and he said that we should try to use Facebook Messenger as the name of the channel 🤔

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md [hunk 3]
```diff
@@ -145,6 +145,18 @@ Conversation API supports Facebook Messenger and allows sending messages from Fa
...
+>
+> This condition impacts you if:
+> - Your app points to messenger bot which is created for Europe page.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support/conversation-facebook-messenger.md:153]: European

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md [hunk 4]
```diff
@@ -145,6 +145,18 @@ Conversation API supports Facebook Messenger and allows sending messages from Fa
...
+> - Your app points to messenger bot which is created for a page that has European admins. 
+> - You send a message to people in Europe.
```
Comment (matokeni) [inline docs/conversation/conversation-channel-support/conversation-facebook-messenger.md:155]: customers :D

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md [hunk 5]
```diff
@@ -145,6 +145,18 @@ Conversation API supports Facebook Messenger and allows sending messages from Fa
 
 - You should receive callbacks with information that message has been delivered and read. The channel is now configured.
 
+> 🚧 Warning (until 16th December)
```
[the remark this answers is not in the document]
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support/conversation-facebook-messenger.md:148]: yeap you are absolutely right, thx for this catch.

[the remark this answers is not in the document]
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support/conversation-facebook-messenger.md]: Changed

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md (hunk 1 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support/conversation-facebook-messenger.md]: Changed.

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md (hunk 2 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support/conversation-facebook-messenger.md]: Changed.

File: docs/conversation/conversation-channel-support/conversation-facebook-messenger.md (hunk 4 above)
Comment (Filip-Kus) [reply docs/conversation/conversation-channel-support/conversation-facebook-messenger.md]: Changed.

**Pull request merged**